### PR TITLE
chore: Adding unit tests for custom Webui URL

### PR DESCRIPTION
**Pull request by gatici**
Adding unit test to validate that custom Webui URL is successfully set.

### CODE REVIEW

File: factory/amf_config_test.go
Comment (gab-arrobo) [inline factory/amf_config_test.go:1]: Please add copyright/license header to file. Thanks!

Comment (gatici) [reply factory/amf_config_test.go:1]: Added.

File: factory/amf_config_test.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// SPDX-FileCopyrightText: 2024 Open Networking Foundation <info@opennetworking.org>
+// Copyright 2019 free5GC.org
```
Comment (gab-arrobo) [inline factory/amf_config_test.go:2]: Is not this a file that you are creating? If so, should it only have a "Canonical" copyright?

Comment (gatici) [reply factory/amf_config_test.go]: You are right. I only added "Canonical" copyright now.

File: amfTest/amfcfg_with_custom_webui_url.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Canonical Ltd.
```
Comment (gab-arrobo) [inline amfTest/amfcfg_with_custom_webui_url.yaml:2]: To use REUSE default format (https://reuse.readthedocs.io/en/stable/usage.html#annotate)
```suggestion
# SPDX-FileCopyrightText: 2024 Canonical Ltd.
```

File: factory/amf_config_test.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2024 Canonical Ltd.
```
Comment (gab-arrobo) [inline factory/amf_config_test.go:2]: To use REUSE default format (https://reuse.readthedocs.io/en/stable/usage.html#annotate)
```suggestion
// SPDX-FileCopyrightText: 2024 Canonical Ltd.
```

**Review by gab-arrobo — APPROVED** [review]
+1

**Pull request merged**